developer_setup.md

```diff
@@ -43,32 +18,25 @@ As per 20+, with the following changes:
   sudo dnf -y install postgresql-devel postgresql-server # PostgreSQL Database server and to build 'pg' Gem
   sudo dnf -y install libxml2-devel libxslt-devel patch  # For Nokogiri Gem
   sudo dnf -y install gcc-c++                            # For event-machine Gem
+  sudo dnf -y install openssl-devel                      # For rubygems
   ```
 
-#### CentOS 7
-
-* Enable EPEL
+* Enable Memcached
 
   ```bash
-  sudo rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+  sudo systemctl enable memcached
+  sudo systemctl start memcached
   ```
 
-* Install packages
-
-  ```bash
-  sudo yum -y install git-all memcached postgresql-devel postgresql-server \
-  libxml2-devel libxslt-devel gcc-c++ patch
-  ```
-* Initialize postgresql and enable services
+* Configure PostgreSQL
 
   ```bash
+  sudo passwd postgres <new_password>
   sudo postgresql-setup initdb
-  sudo su postgres -c "echo local all all trust > /var/lib/pgsql/data/pg_hba.conf"
-  sudo systemctl start memcached
-  sudo systemctl enable memcached
-  sudo systemctl start postgresql
+  sudo sed -i.bak 's/\(^local\s*\w*\s*\w*\s*\)\(peer$\)/\1trust/' /var/lib/pgsql/data/pg_hba.conf
```

Comment:
For our actual appliances, for the local unix file socket we use trust. If the user has access to the local machine. Then they probably can reconfigure the database how ever they want.
For network transfer (even from 127.0.0.1) we use md5 passwords.

Are we sure that the local line is in the text file?

```bash
grep -q '^local\s' pg_hba.conf || echo "local all all trust" > pg_hba.conf
sudo sed -i.bak 's/\(^local\s*\w*\s*\w*\s*\)\(peer$\)/\1trust/' pg_hba.conf
```